react-multiple-components/my-app/src/RotatingBanner.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import Next from './Next';
+import Prev from './Prev';
+import Indicator from './Indicator';
+import Banner from './Banner';
+
+type Props = {
+  itemsRotating: string[];
+};
+
+export function RotatingBanner({ itemsRotating }: Props) {
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [indicatorIndex, setIndicatorIndex] = useState(0);
```

Comment:
I don't think you need both of these. They are both updated identically, so they should always both have the same value. So you could just use one of them.